constants/Colors.js

```diff
@@ -11,4 +13,6 @@ export default {
   warningText: '#666804',
   noticeBackground: tintColor,
   noticeText: '#fff',
-};
+  primaryColor: appJSON.expo.primaryColor,
```

Comment:
It would be useful to have these names more color-specific such as redPrimary.

Reply:
Maybe just comment the color next to it then.

Reply:
We discussed this on the webex since then. We agreed having it generic works better in case we ever change our mind on what the primary color should be or if we add a theme feature.